### PR TITLE
Feature/em 1911 pdf viewer service unit test increase coverage

### PR DESCRIPTION
**Pull request by yyaremchuk**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: projects/media-viewer/src/lib/annotations/services/annotation-api/annotation-api.service.ts [hunk 1]
```diff
@@ -29,17 +30,15 @@ export class AnnotationApiService {
     return this.httpClient
       .post<AnnotationSet>(this.annotationSetsFullUrl, body, { observe: 'response' , withCredentials: true })
       .pipe(
-        map(response => response.body),
-        catchError(() => [])
```
Comment (benouaer) [inline projects/media-viewer/src/lib/annotations/services/annotation-api/annotation-api.service.ts:33]: why remove the error handling?

File: projects/media-viewer/src/lib/viewers/pdf-viewer/side-bar/bookmarks/bookmarks.component.spec.ts [hunk 2]
```diff
@@ -22,18 +47,19 @@ describe('BookmarksComponent', () => {
...
     TestBed.configureTestingModule({
       declarations: [ BookmarksComponent ],
+      providers: [
+        provideMockStore({ initialState })
+      ],
       imports: [
-        StoreModule.forFeature('media-viewer', reducers),
-        StoreModule.forRoot({})
+        // StoreModule.forFeature('media-viewer', reducers),
+        // StoreModule.forRoot({})
```
Comment (benouaer) [inline projects/media-viewer/src/lib/viewers/pdf-viewer/side-bar/bookmarks/bookmarks.component.spec.ts:55]: I assume these imports are no longer needed

File: projects/media-viewer/src/lib/viewers/pdf-viewer/side-bar/bookmarks/bookmarks.component.spec.ts [hunk 3]
```diff
@@ -87,5 +113,95 @@ describe('BookmarksComponent', () => {
...
+    expect(component.editableBookmark).toEqual(mockId);
+  });
+
+  describe('goToBookmark', () => {
+    const mockBookmark = {
+      pageNumber: 0,
+      yCoordinate: 100
+    };
+
+    it('should emit goToDestination event no ratation', () => {
```
Comment (benouaer) [inline projects/media-viewer/src/lib/viewers/pdf-viewer/side-bar/bookmarks/bookmarks.component.spec.ts:129]: I assume you mean rotation

File: projects/media-viewer/src/lib/viewers/pdf-viewer/side-bar/bookmarks/bookmarks.component.ts [hunk 4]
```diff
@@ -118,6 +117,7 @@ export class BookmarksComponent implements OnInit, OnDestroy {
 
     this.goToDestination.emit([
       bookmark.pageNumber,
+      // @TODO: Should the name be hard coded?
```
Comment (benouaer) [inline projects/media-viewer/src/lib/viewers/pdf-viewer/side-bar/bookmarks/bookmarks.component.ts:120]: yes, it's more of a type than a name really

File: projects/media-viewer/src/lib/viewers/pdf-viewer/side-bar/bookmarks/bookmarks.component.spec.ts (hunk 2 above)
Comment (yyaremchuk) [reply projects/media-viewer/src/lib/viewers/pdf-viewer/side-bar/bookmarks/bookmarks.component.spec.ts:55]: Removed them.

File: projects/media-viewer/src/lib/viewers/pdf-viewer/side-bar/bookmarks/bookmarks.component.ts (hunk 4 above)
Comment (yyaremchuk) [reply projects/media-viewer/src/lib/viewers/pdf-viewer/side-bar/bookmarks/bookmarks.component.ts:120]: oh, I see

File: projects/media-viewer/src/lib/viewers/pdf-viewer/side-bar/bookmarks/bookmarks.component.spec.ts (hunk 3 above)
Comment (yyaremchuk) [reply projects/media-viewer/src/lib/viewers/pdf-viewer/side-bar/bookmarks/bookmarks.component.spec.ts:129]: yes, typo. Updated.

File: projects/media-viewer/src/lib/annotations/services/annotation-api/annotation-api.service.ts (hunk 1 above)
Comment (yyaremchuk) [reply projects/media-viewer/src/lib/annotations/services/annotation-api/annotation-api.service.ts:33]: that code does not handle anything. also there should be catchError operator in effects to handle errors in the UI.

**Review by benouaer — DISMISSED** [review]

File: projects/media-viewer/src/lib/annotations/services/annotation-api/annotation-api.service.ts (hunk 1 above)
Comment (yyaremchuk) [reply projects/media-viewer/src/lib/annotations/services/annotation-api/annotation-api.service.ts]: Added back.

**Review by benouaer — APPROVED** [review]

**Review by benouaer — APPROVED** [review]

**Pull request merged**